projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.scss

```diff
@@ -30,6 +31,22 @@
       margin-left: 0;
     }
   }
+
+  .extraction-menu, .anatomical-structures-menu {
+    margin: 0;
+
+    .expansion-header {
+      padding: 0;
+      font-weight: bold;
+      height: 3rem;
+      margin-top: 0 !important;
```

Comment:
The need for important here indicates that something is wrong! From what I can tell from the preview
```
:not(hr) {
    margin-left: 1.5rem;
    margin-right: 1.5rem;
  }
  *:first-child {
    margin-top: 1.5rem;
  }
```
needs to be made specific to the top element by adding `& > ` before each selector